### ISSUE TITLE
Open/Link tracking notifications not functioning correctly

### ISSUE THREAD

**Issue by GM-Polyakov** [nylas/N1#2986]
Events show up in the notification bar but are not reflected by the icons in the thread itself.

**Comment by GM-Polyakov** [comment]
Also, sometimes links don't have the clicked notification dot load at all (empty or full).

**Comment by jackiehluo** [comment]
@GM-Polyakov Links that are explicitly hyperlinked?

**Comment by GM-Polyakov** [comment]
@jackiehluo That's correct. For example, if you link "my.timekit.io/nylas" it will fail to notify 100% of the time.
